p2p/conn/connection.go

```diff
@@ -228,7 +228,9 @@ func (c *MConnection) OnStart() error {
 	c.quitSendRoutine = make(chan struct{})
 	c.doneSendRoutine = make(chan struct{})
 	c.quitRecvRoutine = make(chan struct{})
+	c.Logger.Debug("Starting sendRoutine")
```

Comment:
All logs currently in this file will be deleted. They were originally included to provide insight into the status of the send queue and various other components.